addons/mail/static/src/new/composer/composer.js

```diff
@@ -421,7 +421,12 @@ export class Composer extends Component {
     onFocusin() {
         this.props.composer.isFocused = true;
         if (this.props.composer.thread) {
-            this.threadService.markAsRead(this.props.composer.thread);
+            const isChatterThread = this.props.composer.thread.type === "chatter";
```

Comment:
Not for this PR it would be nice to remove this thread type field that makes no sense.
Basically condition should be on model being channel (or mailbox), the rest being what is considered "chatter".